docs/docs/01-start-here/02-installation.md

```diff
@@ -2,44 +2,52 @@
 id: installation
 title: Installation
 keywords: [Wing installation, installation, Wing toolchain]
-slug: /
 ---
 
-In this section you will install Wing on your system and run your first wing application.
+## Prerequisite
 
-## Prerequisites
+* [Node.js](https://nodejs.org/en/) v20 or later
 
-To install Wing, you will need the following setup:
+## Install
 
-* [Node.js](https://nodejs.org/en/) (>= 20)
-* [VSCode](https://code.visualstudio.com/) (not required, but currently supported with an [extension](#wing-vscode-extension))
-
-## Wing Toolchain
-
-The Wing Toolchain is distributed via [npm](https://www.npmjs.com/):
+The Wing CLI is distributed via [npm](https://www.npmjs.com/package/winglang):
 
 ```sh
 npm install -g winglang
 ```
 
 Verify your installation:
 ```
-wing --version
+wing -V
 ```
 
-To install the Wing VSCode extension, [download](https://marketplace.visualstudio.com/items?itemName=Monada.vscode-wing) it from the VSCode Marketplace. It is distributed via the VSCode Marketplace.
+## IDE Extension
+
+Wing has extended support for two IDEs. They provide syntax highlighting, completions, go-to-definition, etc. and embedded Wing Console support:
 
+- [VSCode](https://marketplace.visualstudio.com/items?itemName=Monada.vscode-wing) - Official extension
+- [IntelliJ](https://plugins.jetbrains.com/plugin/22353-wing) - Community extension
+
+To use Wing in other IDEs, there are a few tools available to help:
+
+- Language server - Running `wing lsp` serves the official language server
+- [TextMate grammar](https://github.com/winglang/wing/blob/main/apps/vscode-wing/syntaxes/wing.tmLanguage.json) - For syntax highlighting
+- [tree-sitter grammar and queries](https://github.com/winglang/wing/tree/main/libs/tree-sitter-wing) -For syntax highlighting and more
 
```

Comment:
I would also add another community tool for the GitHub syntax highlighting by @eladcon